README.md

```diff
@@ -30,20 +30,41 @@ Instructions on setting up the project and getting it running on a local machine
 - Double-check your CHT test instance is running
 - Clone or fork the test data generator repository
 - Install and build the project by running `npm ci` in the project root folder
-- Design the test data in a custom JavaScript file. See section [Designing Test Data](#designing-test-data).
-- Build, generate data and upload by running `npm run generate *path_to_your_custom_design_file*`
+- Design the test data in a custom JavaScript file. See the section [Designing Test Data](#designing-test-data).
+- Build, generate data, and upload by running `npm run generate *path_to_your_custom_design_file*`
 
 ### Install it globally
 Another option is to install the tool globally:
 - Install package dependencies and build the project `npm ci`
 - Run `npm install -g` in the project root folder
-- Build, generate and upload data by running `tdg <path_to_your_custom_design_file>`.
+- Build, generate, and upload data by running `tdg <path_to_your_custom_design_file>`.
 
 ### Use it with Docker
 The tool is also available in Docker:
 - Create the image `docker build -t test-data-generator .`
 - Run the container `docker run --rm -it -v <folder_path_of_your_design_file>:/app/test-data -e COUCH_URL=<test_instance_CouchDB_URL> -e FILE=<your_design_file> test-data-generator`
 
+
+### Quick Start 
+
+This tool provides a design that will populate your CHT instance with sample data that touches or triggers most Default Configuration workflows.
+This guide assumes that you have installed the `test-data-generator` as a global dependency.
```

Comment:
Thanks for adding this!  I'd not done this and `tdg` was giving me `command not found: tdg`. I needed to more carefully read your steps :sweat_smile: